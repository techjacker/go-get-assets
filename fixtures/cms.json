{
  "nophotos": "here",
  "mapofphotos": {
    "photourl": "http://gdrive.com/diff.jpg",
    "noaphoto": "sdfsd"
  },
  "clients": [{
    "name": "Samsung",
    "photourl": "http://gdrive.com/city.jpg"
  }, {
    "name": "Sony",
    "photourl": "http://gdrive.com/sf.jpg"
  }],
  "projects": [{
    "descriptionsub": "Launch Project",
    "descriptionsublink": "http://themummyofmulberryavenue.com",
    "photourl": "http://gdrive.com/water.jpg, http://gdrive.com/highway.jpg, http://gdrive.com/freedom.jpg, http://gdrive.com/mountain.jpg, http://gdrive.com/traffic.jpg"
  }, {
    "descriptionsub": "Launch Project",
    "descriptionsublink": "http://themummyofmulberryavenue.com",
    "photourl": "http://gdrive.com/water.jpg, http://gdrive.com/highway.jpg, http://gdrive.com/freedom.jpg, http://gdrive.com/mountain.jpg, http://gdrive.com/traffic.jpg"
  }]
}