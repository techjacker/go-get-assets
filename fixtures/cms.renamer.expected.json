{
  "nophotos": "here",
  "mapofphotos": {
    "photourl": "/images/diff.jpg"
  },
  "clients": [{
    "name": "Samsung",
    "photourl": "/images/city.jpg"
  }, {
    "name": "Sony",
    "photourl": "/images/sf.jpg"
  }],
  "photourl": "/images/freedom.jpg,/images/traffic.jpg"
}